dev-client/src/components/projects/EditProjectInstructionsScreen.tsx

```diff
@@ -0,0 +1,133 @@
+import {Button, Heading, HStack, Spacer, Box} from 'native-base';
+import {Formik} from 'formik';
+import {useTranslation} from 'react-i18next';
+import {useDispatch} from 'terraso-mobile-client/model/store';
+import {SiteNoteForm} from 'terraso-mobile-client/components/siteNotes/SiteNoteForm';
+import {ProjectUpdateMutationInput} from 'terraso-client-shared/graphqlSchema/graphql';
+import {
+  updateProject,
+  Project,
+} from 'terraso-client-shared/project/projectSlice';
+import {HorizontalIconButton} from 'terraso-mobile-client/components/common/Icons';
+import ConfirmModal from 'terraso-mobile-client/components/common/ConfirmModal';
+import {ScreenScaffold} from 'terraso-mobile-client/screens/ScreenScaffold';
+import {useNavigation} from 'terraso-mobile-client/screens/AppScaffold';
+import {Keyboard, KeyboardAvoidingView, Platform} from 'react-native';
+
+type Props = {
+  project: Project;
+};
+
+export const EditProjectInstructionsScreen = ({project}: Props) => {
+  const {t} = useTranslation();
+  const navigation = useNavigation();
+  const dispatch = useDispatch();
+
+  const handleUpdateProject = async (content: string) => {
+    Keyboard.dismiss();
+    try {
+      const projectInput: ProjectUpdateMutationInput = {
+        id: project.id,
+        siteInstructions: content,
+      };
+      const result = await dispatch(updateProject(projectInput));
+      if (result.payload && 'error' in result.payload) {
+        console.error(result.payload.error);
+        console.error(result.payload.parsedErrors);
+      }
+    } catch (error) {
+      console.error('Failed to update project:', error);
+    } finally {
+      navigation.pop();
+    }
+  };
+
+  const handleDelete = async (
+    setSubmitting: (isSubmitting: boolean) => void,
+  ) => {
+    Keyboard.dismiss();
+    setSubmitting(true);
+    await handleUpdateProject('');
+    setSubmitting(false);
+  };
+
+  return (
+    <ScreenScaffold BottomNavigation={null} AppBar={null}>
+      <KeyboardAvoidingView
+        behavior={Platform.OS === 'ios' ? 'padding' : 'height'}
```

Comment:
same questions about reusing comments and refactoring as with site notes screen.